### PR TITLE
Our music

### LINKED ISSUE #25: OurMusic - Instruments
Our Music

### PR DESCRIPTION
**Pull request by Yehezkiel01**
Recreated from #34 (originally created by @RussellDash332 )

Final things to be done :

Update the pictures of kolintang, gendang, and gitar
Change the placeholder video to the NUS Angklung spotify playlist

Close #25

### CODE REVIEW

File: src/routes/our-music/index.svelte [hunk 1]
```diff
@@ -47,3 +100,119 @@
...
+        <b>Our Playlist</b>
+    </h2>
+</center>
+<br />
+<div class="iframe-container iframe-container-16x9">
+    <iframe
+        title="Placeholder video for Spotify Playlist"
+        width="560"
+        height="315"
+        src="https://www.youtube.com/embed/dQw4w9WgXcQ"
```
Comment (Yehezkiel01) [inline src/routes/our-music/index.svelte]: For playlist, we can use this for now https://youtube.com/playlist?list=PLac9vFnrjsKMJqs_HuwKg7EbwU9uFTsDR

File: src/routes/our-music/index.svelte [hunk 2]
```diff
@@ -21,19 +62,31 @@
...
+        height: 30%;
+        object-fit: contain;
+    }
 </style>
 
 <!-- <h4>Our Music</h4> -->
 
+<center>
+    <h2>
+        <b>Our Performances</b>
```
Comment (Yehezkiel01) [inline src/routes/our-music/index.svelte:74]: The page is getting longer, I would suggest to remove other performances and only keeping the 2019 Performance

**Review by Yehezkiel01 — COMMENTED** [review]
LGTM!

@RussellDash332 minor changes requested. You can merge after that 😉

File: src/routes/our-music/index.svelte (hunk 2 above)
Comment (RussellDash332) [reply src/routes/our-music/index.svelte:74]: Will do. Let me commit again.

File: src/routes/our-music/index.svelte [hunk 3]
```diff
@@ -65,10 +65,10 @@
     .iframe-container-16x9 {
         padding-bottom: calc(9 / 16 * 100%);
     }
-    .caption {
+    /* .caption {
```
Comment (Yehezkiel01) [inline src/routes/our-music/index.svelte]: In general, it's a bad practice to keep the unused code as comments. Most likely we'll never use that again. So, let's remove all commented codes for now. In the worst case, we can always check git history for the code.

File: src/routes/our-music/index.svelte [hunk 4]
```diff
@@ -47,3 +100,142 @@
...
+        height="315"
+        src="https://youtube.com/embed/playlist?list=PLac9vFnrjsKMgSrV3XdcHyMKeyX17BP8M"
+        frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media;
+        gyroscope; picture-in-picture"
+        allowfullscreen />
+</div>
+
+<center>
+    <h3>Spotify Playlist</h3>
```
Comment (Yehezkiel01) [inline src/routes/our-music/index.svelte]: Not exactly Spotify Playlist, can we named it Other Playlist Instead?

Comment (RussellDash332) [reply src/routes/our-music/index.svelte]: Done.

**Review by RussellDash332 — APPROVED** [review]
LGTM and you!

**Pull request merged**